spec.emu

```diff
@@ -150,7 +150,7 @@ contributors: Ujjwal Sharma, Younies Mahmoud
             1. Append _field_ as the last element of _list_.
           1. Set _durationFormat_.[[Fields]] to _list_.
         1. Else if IsArray(_fields_) is *true*, then
-          1. Set _fields_ to ? CreateListFromArrayLike(_fields_).
+          1. Set _fields_ to ? CreateListFromArrayLike(_fields_, « String »).
```

Comment:
Do you want this to produce an error if the field isn't a string, i.e `[{ toString() { return "year" } }]`, or do you want to coerce to string via `ToString`? I'm not certain if there's precedent in `Intl`, but `Object.defineProperty` uses `ToString()`. If throwing a `TypeError` for non-string values for fields is what you want, then this looks fine to me.